src/content/docs/telemetry-data-platform/ingest-manage-data/manage-data/query-limits.mdx

```diff
@@ -0,0 +1,207 @@
+```
+  </Collapser>
+</CollapserGroup>
+
+## Limit metrics
+
+These metrics, used in the dashboard queries above, can hone in on a single limit or resource. Or, with the help of `FACET limitName or Resource` provide a view across all your limits.
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id=""
+    title="newrelic.resourceConsumption.limitValue"
+  >
+`limitValue` allows you to see the setting for a limit by `limitName` and understand more about what resource is linked to this limit. The following examples use the limit value metric in the query:
+
+- Example for Metric API requests per minute.
+```
+FROM Metric select latest(newrelic.resourceConsumption.currentValue) where limitName = 'Metric API requests per minute'  limit 200
+```
+- To show all limits, add `FACET limitName` and consider grouping by `limitTimeInterval`.
+
+```
+FROM Metric select latest(newrelic.resourceConsumption.currentValue) WHERE limitTimeInterval = '1 minute' FACET limitName  limit 200
+```
+
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id=""
+    title="newrelic.resourceConsumption.currentValue"
+  >
+`currentValue` shows you how much of a given resource you’re currently consuming. To get a better glimpse into how our systems are viewing your consumption,  use a `rate()` function with the time period that aligns with the `limitWindow`. Limit 200.
+
+
+- Example for Metric API request per minute:
+```
+FROM Metric select rate(sum(newrelic.resourceConsumption.currentValue),1 minute) where limitName = 'Metric API requests per minute' “  limit 200
+```
+- To show all limits, add `FACET limitName` and consider grouping by `limitTimeInterval`.
+```
+FROM Metric select rate(sum(newrelic.resourceConsumption.currentValue),1 minute) where limitTimeInterval = '1 minute' FACET limitName “  limit 200
+```
+
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id=""
+    title="newrelic.resourceConsumption.impact"
+  >
+`impact` lets you know for any given resource what impact limit events are having. Zeros mean you are not currently impacted.
+
+The most granular we have is `dataType`. It is possible for multiple instances of `limitName` to impact a single type, such as Metric RPM and DPM. If we know, we will display `limitName`.
+
+```
+From Metric select rate(sum(newrelic.resourceConsumption.impact), 1 minute) facet dataType, resource, impact, limitName   TIMESERIES  limit 200
+```
+  </Collapser>
+</CollapserGroup>
+
+### Metric attributes
+
+Attributes on `newrelic.resourceConsumption.limitValue`  and `newrelic.resourceConsumption.currentValue`:
+- `limitName` - The Name of the limit for the metric data, i.e RPM Metric API.
+- `dataType` - What kind of data the metric is tracking, i.e Metric, Log, or APM.
+- `Resource` - What resource is being consumed, i.e. Requests, or DPM.
+- `limitTimeInterval` - What time window this resource is evaluated for limiting.
+- `account` -  
```

Comment:
This should be `consumingAccountId`  It is:

The New Relic account where the resource is being consumed.